### ISSUE TITLE
Unstable Sorting in EditorProvider#getToolboxGroups() Function

### ISSUE THREAD

**Issue by mythsman** [nocode-js/sequential-workflow-editor#46]
Although the sorting of `EditorProvider#getToolboxGroups` itself is stable, when adding steps using `DefinitionModelBuilder#steps`, the originally ordered steps get regrouped based on `model.type` . After that, using `Object.values(this.definitionModel.steps)` does not guarantee the order.

**Comment by b4rtaz** [comment]
Hello @mythsman,

I'm not sure if I understand correctly. The `getToolboxGroups` method returns groups and steps in alphabetical order. For that, the `localeCompare` method is used.

```ts
	public getToolboxGroups(): ToolboxGroup[] {
		const stepModels = Object.values(this.definitionModel.steps);
		...
		categories.forEach((category: string | undefined) => {
			...
			groupSteps.sort((a, b) => a.name.localeCompare(b.name));
			groups.push({
				name,
				steps: groupSteps
			});
		});
		groups.sort((a, b) => a.name.localeCompare(b.name));
		return groups;
	}
```

Do you want to keep the order from the model builder?

**Comment by b4rtaz** [comment]
@mythsman please check the [0.14.3](https://github.com/nocode-js/sequential-workflow-editor/releases/tag/v0.14.3) version. I added both features.

**Comment by b4rtaz** [comment]
@mythsman fixed in [0.14.4](https://github.com/nocode-js/sequential-workflow-editor/releases/tag/v0.14.4). Thanks!
